src/admin/seeds/dashboard/dashboard.html

```diff
@@ -78,12 +78,12 @@
                 </div>
               </div>
             </div>
-            <div class="featureContainer" if.bind="selectedSeed.projectTokenBalance >= selectedSeed.seedRemainder && (selectedSeed.isClosed || selectedSeed.claimingIsOpen)">
+            <div class="featureContainer" if.bind="selectedSeed.projectTokenBalance >= selectedSeed.seedRemainder && (selectedSeed.isClosed || selectedSeed.maximumReached || selectedSeed.isDead)">
```

Comment:
Why are you not just using `seed.claimingIsOpen`?

Reply:
These are the conditions that are used in contract to check if `retrieveSeedTokens` can be invoked or not.

Reply:
ah, i get it.